src/main/scala/tech/cryptonomic/conseil/generic/chain/DataPlatform.scala

```diff
@@ -25,7 +25,7 @@ class DataPlatform(operationsMap: Map[String, DataOperations]) {
     * @return query result as a option[map]
     * */
   def queryWithPredicates(platform: String, tableName: String, query: Query)
-    (implicit ec: ExecutionContext): Option[Future[List[Map[String, Option[Any]]]]] = {
+    (implicit ec: ExecutionContext): Option[Future[List[QueryResponse]]] = {
```

Comment:
I guess we can improve consistency in the calls if we `sequence` the result here to return a
`Future[Option[List[QueryResponse]]]`

On a personal note, I find this more to my taste than the other way round

Reply:
sure, changed